docs/support/upgrade.md

```diff
@@ -225,5 +225,3 @@ publicApi.post('/examples', async (ctx) => {
   await examples.doc(name).set({ name, description });
 });
 ```
-
-> The nitric.yaml and api.yaml are not required.
```

Comment:
maybe the api.yaml is not required?

Reply:
@raksiv could you update with the new nitric.yaml details

Reply:
Added new yaml details, and a note that the api.yaml is not required.